src/streaming/controllers/PlaybackController.js

```diff
@@ -257,7 +257,8 @@ function PlaybackController() {
      */
     function getStreamStartTime(ignoreStartOffset) {
         let presentationStartTime;
-        let startTimeOffset = !ignoreStartOffset ? parseInt(URIQueryAndFragmentModel(context).getInstance().getURIFragmentData().s, 10) : NaN;
+        let fragData = URIQueryAndFragmentModel(context).getInstance().getURIFragmentData();
+        let startTimeOffset = !ignoreStartOffset ? parseInt(fragData.s || fragData.t, 10) : NaN;
```

Comment:
if fragData.s value is set to 0 it will not set and use the other side of the || and look for fragData.t ....I believe. 

Keep seeing this happen all over the code with things that could be set to 0 and we use something || something else.